client/src/pages/Dashboard/UserDetails/UserDetails.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import Card from '@mui/material/Card';
+import CardMedia from '@mui/material/CardMedia';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import Grid from '@mui/material/Grid';
+import Divider from '@mui/material/Divider';
+import useStyles from './useStyles';
+import { Box } from '@mui/material';
+import Rating from '@mui/material/Rating';
+import StarIcon from '@mui/icons-material/Star';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+interface UserDetailsProps {
+  user: {
+    id: number;
+    image: string;
```

Comment:
Question - wouldn't this be the `Profile` interface?

Reply:
Yes! Now  I am using the user from mock users. Thanks

Reply:
Hey @OsmanRana, chiming in to clarify this a little bit: in our design, the `user` model/interface contains account-related information such as email/password. We use the *profile* interface to store personal information about the user, exactly what this interface contains! That's why Ethan suggested to call this interface `profile` instead of `user`. 

Let's rename this to `profile` (and other places where we use `user` instead of `profile`) so it doesn't cause any confusion, also let's move it into `client/src/interface` folder while we are at it👍🏼

Reply:
user renamed to profile and added a new file named Profile in client/src/interface